docusaurus/docs-internals/editor-controls/data-controls/editableSelect.md

```diff
@@ -0,0 +1,406 @@
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `choices`          | `Choices`                                                                                                                                                                                  |    -    | `Choices` is an array of `Choice` objects that are directly provided to the component.<br/><br/>**`Choice: { title: string; value: string; allowRemove?: boolean; allowEdit?: boolean; allowDuplicate?: boolean; }`**<br/><br/> Each `Choice` object defines the following properties:<br/>`allowRemove?` - Allows to remove the `Choice`.<br/>`allowDuplicate?` - Allows to duplicate the `Choice`.<br/>`allowEdit?` - Allows to edit the `Choice`.<br/>`title` - A string representing the display title of the choice.<br/>`value` - A string value representing the choice. This value is returned when the choice is selected. |
+| `onChange`         | `(action: OnChangeAction) => void;`                                                                                                                                                        |    -    | Changes the value or the choices array, depends on action. Receives `OnChangeAction` which is `{ type: OnChangeActionTypes; payload: string; }` and `OnChangeActionTypes` is `enum OnChangeActionTypes { remove = "remove", duplicate = "duplicate", edit = "edit", change = "change", reset = "reset" }`                                                                                                                                                                                                                                                                                                                           |
```

Comment:
for `onChange`:

1. Write that his type is `function` and then in description describe more explicitly.
2. Also prettify the description because now it not readable
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/6955f287-24c2-4f6d-b281-7baa0ececca9)